python_settings/__init__.py

```diff
@@ -121,6 +121,8 @@ def __getattr__(self, item):
                     "LazySetting(MyCustomClass, [params])"
                     "Exception: %s - %s" % (type(ex), ex.__repr__())
                 )
+        if get_attr is empty:
+            raise AttributeError
```

Comment:
cool! but instead of just raising the exception, can you provide more information to the user of what is happening?
Something like: "settings object has no attribute 'WRONG_ATTRIBUTE'"

And also can you add a unit test to add coverage to your new line?

Reply:
the latest commit should take care of this